### DISCUSSION TITLE
Community Chat - Tell us your thoughts

### DISCUSSION THREAD

**Discussion by MikeCodesDotNET** [AvaloniaUI/Avalonia#11369] [Polls]
There’s been some discussion on our community Telegram chat that we might be better off moving the community chat elsewhere. 

We’re interested to hear you’re thoughts on where you’d prefer to interact with the community. 

**I’ll leave the poll open for a few days.**

**Comment by wieslawsoltes** [comment]
Telegram is best option, works great on all platforms especially mobile and web for me.

**Comment by Tamnac** [comment]
The telegram is bridged with gitter/matrix, which is what I use, but it's a poor experience. If nothing else, I hope you improve the bridge.

**Comment by kyuranger** [comment]
I think GitHub Discussions should be kept. This is one of the important discussion forums. Telegram is banned in some countries. Using Telegram only will affect normal communication.

**Comment by cotti** [comment]
Best option would be something based on Free Software, so Matrix (which Gitter has integrated itself into) is the natural choice. Keeping the Github Discussions open is important as well, since it can be accessed and at least read by anyone. Telegram is bridged at least, and has the client released as Free Software.

Honestly, hopefully anything but Discord, which is not only more user and privacy-hostile than the others, but also walls its content, barring users in need.

**Comment by MikeCodesDotNET** [comment]
GitHub discussions aren’t going anywhere.

**Comment by Flithor** [comment]
An other proposal: [Stackoverflow Chat  rooooooooooom](https://chat.stackoverflow.com/)!
At least it can be confirmed that almost every programmer has a account on StackOverflow ;)

**Comment by M0n7y5** [comment]
Telegram cannot be used without phone number.
